index.html

```diff
@@ -9,20 +9,23 @@
     <link rel="stylesheet" href="./style.css">
     <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
     <!-- Header -->
     <section class="header">
         <header class="nav">
             <nav>
-                <img class="menu" src="./assets/ic_menu.svg" alt="hamburger menu">
-                <ul>
+                <img class="menu" id="hamburger" src="./assets/ic_menu.svg" alt="hamburger menu">
+                <ul id="ul">
                     <li><a class="underline" href="#Home">Home</a></li>
                     <li><a href="#project">Portfolio</a></li>
                     <li><a href="#about">About</a></li>
                     <li><a href="#contact">Contact</a></li>
+                    <div class="mob-abs-dot"></div>
                     <div class="mob-abs-2"></div>
+                    <img id="cross" class="menu-cross" src="./assets/cross.svg" alt="cross">
                 </ul>
             </nav>
         </header>
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/201690565-b968ea51-b050-4e83-88f9-27430a022d4a.PNG)

- [ ] When i click on any of the nav link in the desktop version then they work as expected but after clicking these link get vanish. Kindly fix this issue 👍

Reply:
You are not reviewing the mobile menu. Please keep your focus on a given task. You are talking about optional features and wasting my review.

![additional](https://user-images.githubusercontent.com/114587346/201693603-d52eb0a0-e5ed-4b5c-ac1b-cc77a83eca92.png)